.github/workflows/artifacts-build.yml

```diff
@@ -119,31 +118,42 @@ jobs:
         packages: |
           sphinx
           sphinx-rtd-theme
+    - name: Install Dart
+      uses: dart-lang/setup-dart@v1
```

Comment:
Use hashes for version pinning, see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions

Reply:
We're not doing this anywhere but I'll file an issue.